sycl-fusion/common/CMakeLists.txt

```diff
@@ -1,16 +1,25 @@
-add_llvm_library(sycl-fusion-common
+add_library(sycl-fusion-common
```

Comment:
Can we add a comment for future reference to explain why we're taking the manual approach here?